opt-may-2016/anusha/com/anusha/training/Example2.java

```diff
@@ -0,0 +1,27 @@
+package com.anusha.training;
+
+import java.sql.Date;
+
+public class Example2 {
+
+	public static void main(String[] args) {
+	try{
+		Date d=null;
+		if(d ==null){
+			d.getTime(); 
+		
+		System.out.println(d);
+		
+		}
+		
+		}
+	
+		catch(NullPointerException nullPointer){

```

Comment:
ideally you should not catch null pointer exceptions ever..if there is a null pointer exception..that means there is something extremely wrong in the code.